db/migrate/20190329201346_move_partner_status_to_an_interger_type.rb

```diff
@@ -0,0 +1,68 @@
+class MovePartnerStatusToAnIntergerType < ActiveRecord::Migration[5.2]
```

Comment:
Typo in the name? `Interger` should be `Integer`?

Reply:
rrr. dang.